### PR TITLE
allow disabling redirect/reload strategy for iOS Safari (webview case)

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

File: src/components/SEBankIDSameDeviceButton.tsx [hunk 1]
```diff
@@ -136,7 +144,7 @@ export default function SEBankIDSameDeviceButton(props: Props) {
       setLinks(links);
 
       const androidChrome = mobileOS === 'android' && userAgent?.browser.name === 'Chrome' ? true : false;
-      const redirect = iOSSafari ? window.location.href : 'null';
+      const redirect = (iOSSafari && strategy === 'Reload') ? window.location.href : 'null';
```
Comment (sgryt) [inline src/components/SEBankIDSameDeviceButton.tsx:147]: Not sure why this change is needed ?

Comment (sgryt) [reply src/components/SEBankIDSameDeviceButton.tsx:147]: Ah, to prevent the previous (implicit) always-reload.

**Comment by mickhansen** [comment]
@sgryt Link with appswitch disabled, for test on your iOS: https://deploy-preview-31--criipto-verify-react-storybook.netlify.app/iframe.html?args=loginHint:YXBwc3dpdGNoOnJlc3VtZVVybDpkaXNhYmxl&id=components-authmethodbutton--se-bank-id-same-device-button&viewMode=story

File: src/components/AuthMethodButton/AuthMethodButton.stories.tsx [hunk 2]
```diff
@@ -42,11 +42,22 @@ export default {
...
+  }
+}
+
+
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
 const Template: ComponentStory<typeof AuthMethodButton> = (args, {globals}) => {
-  const loginHint =
-    (args as any).loginHint && btoa(atob((args as any).loginHint)) === (args as any).loginHint ? atob((args as any).loginHint) :
-    (args as any).loginHint;
+  const loginHint = tryBase64Decode((args as any).loginHint);
```
Comment (sgryt) [inline src/components/AuthMethodButton/AuthMethodButton.stories.tsx:60]: Seems the previous ternary has gone AWOL ?
